src/syscheckd/src/file/file.c

```diff
@@ -63,6 +63,7 @@ STATIC void transaction_callback(ReturnTypeCallback resultType,
     char *diff = NULL;
     char *path = NULL;
     char iso_time[32];
+    int sync_operation = -1;
```

Comment:
Maybe we can define the operation type in the sync_protocol_c header as an enum to make it more clear.